src/utils/display.tsx

```diff
@@ -10,14 +10,31 @@ export function displayTokenSymbolOrLink(token: TokenDetails): React.ReactNode |
   return displayName
 }
 
+/**
+ * computeMarketProp
+ * @description returns array of potentially accepted market names by:: SELLTOKEN <SEPARATOR> BUYTOKEN
+ * @param { sellToken, buyToken, acceptedSeparators: string[] }
+ */
 export function computeMarketProp({
   sellToken,
   buyToken,
+  acceptedSeparators = ['-', '/', ''],
```

Comment:
This would trigger false positives for tokens symbols with  `'-/ '` in the symbol, won't it?

Reply:
well it spits out e.g `WETH-DAI` so if the user is searching for market `WETH-D` - `DAI` it wouldn't find it, which is intended, no?